test.js

```diff
@@ -42,41 +42,41 @@ test('execa.stderr()', async t => {
 });
 
 test('stdout/stderr available on errors', async t => {
-	const err = await t.throws(m('exit', ['2']));
+	const err = await t.throwsAsync(m('exit', ['2']));
 	t.is(typeof err.stdout, 'string');
 	t.is(typeof err.stderr, 'string');
 });
 
 test('include stdout and stderr in errors for improved debugging', async t => {
-	const err = await t.throws(m('fixtures/error-message.js'));
+	const err = await t.throwsAsync(m('fixtures/error-message.js'));
 	t.regex(err.message, /stdout/);
 	t.regex(err.message, /stderr/);
 	t.is(err.code, 1);
```

Comment:
Can you also use the new error matching format? https://github.com/avajs/ava/blob/master/docs/03-assertions.md#throwsasyncthrower-expected-message Then you don't need to return the error.

Reply:
Sure thing, will do. In some of the tests this can't be used though, e.g. when `error.stdout|stderr` is being tested, I don't think the new assertion format supports that.

Reply:
Done